projects/storefrontapp-e2e-cypress/cypress/integration/regression/checkout/checkout-as-guest-with-variants.e2e-spec.ts

```diff
@@ -0,0 +1,33 @@
+import { CheckoutConfig } from '@spartacus/storefront';
+import { checkoutAsGuestWithVariantsTest } from '../../../helpers/checkout-as-guest-with-variants';
+
+context('Checkout as guest', () => {
+  before(() => {
+    cy.window().then(win => win.sessionStorage.clear());
+    cy.cxConfig({ checkout: { guest: true } } as CheckoutConfig);
+    configureApparelProduct();
+  });
+  beforeEach(() => {
+    cy.restoreLocalStorage();
+  });
+
+  checkoutAsGuestWithVariantsTest();
+
+  afterEach(() => {
+    cy.saveLocalStorage();
+  });
+
+  after(() => {
+    // login.signOutUser();
+  });
+
+  function configureApparelProduct() {
```

Comment:
Can we move this function to helper, so that it is not duplicated in `checkout-with-variants.e2e-spec.ts`?